pkg/registry/core/service/ipallocator/allocator_test.go

```diff
@@ -326,7 +326,7 @@ func TestNewFromSnapshot(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	r, err := NewCIDRRange(cidr)
+	r, err := NewInMemory(cidr)
```

Comment:
you forgot to replace the `NewCIDRRange` in line 370

> levee: pkg/registry/core/service/ipallocator/allocator_test.go:370:12: undeclared name: NewCIDRRange

Reply:
fixed - I blame the rebase :)